File: server/Models/Review.cs
namespace Monogram.Models;

public class Review
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Name { get; set; }
    public string Email { get; set; }
    public string Title { get; set; }
    public string Body { get; set; }
    public string ImgUrl { get; set; }
    public int ProductId { get; set; }
}

File: client/src/models/Review.js

export class Review {
    constructor(data) {
        this.id = data.id
        this.createdAt = data.createdAt
        this.updatedAt = data.updatedAt
        this.name = data.name
        this.email = data.email
        this.imgUrl = data.imgUrl
        this.title = data.title
        this.body = data.body
        this.productId = data.productId
    }
}